rosdep/python.yaml

```diff
@@ -2516,6 +2516,37 @@ python-pynmea2:
   ubuntu:
     pip:
       packages: [pynmea2]
+python-pypng:
+  arch:
+    pip:
+      packages: [pypng]
+  debian: [python-png]
```

Comment:
This is only available on buster, we need to provide pip entries for the active distributions that don;t have that package: `jessie` and `stretch`